libecho/src/AudioStream.h

```diff
@@ -0,0 +1,127 @@
+#ifndef ECHOCONNECT_AUDIOSTREAM_H
+#define ECHOCONNECT_AUDIOSTREAM_H
+
+#include <QtCore/QThread>
+#include <QtMultimedia/QAudioFormat>
+#include <QtMultimedia/QAudio>
+#include <QtMultimedia/QAudioInput>
+#include <QDebug>
+#include <QMetaMethod>
+
+constexpr int NOTIFY_INTERVAL = 32;
+constexpr double VOLUME = 1.0;
+
+struct AudioStreamInfo {
```

Comment:
It would be good to document that structure (especially notifyInterval and periodSize).

Reply:
Will do, but documentation/comments will be in next PR - I want to merge working code, so you don't need to branch from my branch.